recipes/cuda_nvcc/meta.yaml

```diff
@@ -0,0 +1,71 @@
+{% set name = "cuda_nvcc" %}
```

Comment:
So I would strongly argue for `cuda-nvcc` over the underscore version. This is a general statement on other `cuda-*` packages.

Reply:
The output packages are in the format of `cuda-*`. This is what I'm seeing in the log: `/home/conda/staged-recipes/build_artifacts/linux-64/cuda-nvcc-11.8.89-ha770c72_0.conda` 
The underscore just comes from how components are named internally, so if that is still and issue I can change that here.

Reply:
Yes please change it. It will keep the feedstock names with dashes too.

Reply:
Think the issue is we can't have the packages in `outputs` and the feedstock `name` match

Reply:
This is renamed to cuda-nvcc now. Should the recipe folder also be renamed?

Reply:
Thanks Alex! 🙏

> Should the recipe folder also be renamed?

For consistency yes. Though I don't think we do anything with the folder name

Reply:
Sounds good, moved the recipe to cuda-nvcc directory now.

Reply:
Extreme nitpick, but: To avoid confusion down the line, we should also rename the title of this PR accordingly. Thanks <3

Reply:
Done